images/ose-must-gather.yml

```diff
@@ -9,7 +9,7 @@ content:
 from:
   builder:
   - stream: golang
-  member: openshift-enterprise-base
+  member: openshift-enterprise-cli
```

Comment:
@smunilla can you confirm this is the correct string.  I don't know where it comes from, but I don't see this one in this repo

Reply:
The intention is to refer to https://github.com/openshift/ocp-build-data/blob/openshift-4.1/images/openshift-enterprise-cli.yml

Reply:
filenames matter?